ilastik/ilastik_logging/default_config.py

```diff
@@ -132,85 +128,20 @@ def get_default_config(
-            "lazyflow.operators.classifierOperators": {"level": "INFO"},
-            "lazyflow.operators.opCompressedCache": {"level": "INFO"},
-            "lazyflow.operators.opRelabelConsecutive": {"level": "INFO"},
-            "lazyflow.utility.io_util.RESTfulVolume": {"level": "INFO"},
-            "lazyflow.utility.io_util.tiledVolume": {"level": "INFO"},
-            "lazyflow.operators.opFeatureMatrixCache": {"level": "INFO"},
-            "lazyflow.operators.opConcatenateFeatureMatrices": {"level": "INFO"},
-            "lazyflow.utility.roiRequestBatch": {"level": "INFO"},
-            "lazyflow.utility.bigRequestStreamer": {"level": "INFO"},
-            "ilastik": {"level": "INFO"},
-            "ilastik.clusterOps": {"level": "INFO"},
-            "ilastik.applets": {"level": "INFO"},
-            "ilastik.applets.base.appletSerializer": {"level": "INFO"},
-            "ilastik.applets.dataSelection": {"level": "INFO"},
-            "ilastik.applets.featureSelection": {"level": "INFO"},
-            "ilastik.applets.pixelClassification": {"level": "INFO"},
-            "ilastik.applets.thresholdTwoLevels": {"level": "INFO"},
-            "ilastik.applets.thresholdTwoLevels.ipht": {"level": "INFO"},
-            "ilastik.applets.objectExtraction": {"level": "INFO"},
-            "ilastik.applets.blockwiseObjectClassification": {"level": "INFO"},
-            "ilastik.applets.tracking.conservation": {"level": "INFO"},
-            "ilastik.shell": {"level": "INFO"},
-            "ilastik.shell.projectManager": {"level": "INFO"},
-            "ilastik.shell.gui.ipcManager": {"level": "INFO"},
-            "ilastik.workflows": {"level": "INFO"},
-            "ilastik.widgets": {"level": "INFO"},
-            "ilastik.utility": {"level": "INFO"},
-            "ilastik.utility.exportingOperator": {"level": "INFO"},
-            "ilastik.utility.exportFile": {"level": "INFO"},
-            "workflows": {"level": "INFO"},
-            "volumina": {"level": "INFO"},
-            "volumina.pixelpipeline": {"level": "INFO"},
-            "volumina.imageScene2D": {"level": "INFO"},
-            "volumina.utility.shortcutManager": {"level": "INFO"},
-            # Python doesn't provide a trace log level, so we use a workaround.
-            # By convention, trace loggers have the same hierarchy as the regular loggers, but are prefixed with 'TRACE' and always emit DEBUG messages
+            "OpenGL": {"level": "INFO"},
+            "yapsy": {"level": "INFO"},
+            # Loglevels for our own modules (ilastik, lazyflow, volumina) are in ./logging_config.json.
+            # Note regarding TRACE settings: Python doesn't provide a trace log level, so we use a workaround.
+            # By convention, trace loggers have the same hierarchy as the regular loggers,
+            # but are prefixed with 'TRACE' and always emit DEBUG messages.
             # To enable trace messages, change one or more of these to use level DEBUG
```

Comment:
Whatever this TRACE setting is supposed to do, is maybe better documented in the file where it's used (`logging_config.json`).

Reply:
I was going to move this comment to the json file but json doesn't support comments. I'll move it over anyway I guess and just make it a text field (as [suggested here](https://stackoverflow.com/questions/244777/can-comments-be-used-in-json)).

Reply:
Looks like in our case this is only actually used in a handful modules. I left the corresponding entries and removed all the other ones; plus the various traceLogger instantiations that weren't used across our codebase.